contracts/storage/Storage.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.9;
+import "./Structs.sol";
+
+contract Storage is Structs {
+    uint256 public constant MANAGEMENT_KEY = 1;
+    uint256 public constant ACTION_KEY = 2;
+    uint256 public constant CLAIM_SIGNER_KEY = 3;
+    uint256 public constant ENCRYPTION_KEY = 4;
```

Comment:
useless variable imo, additionally, the value of the variable will not be shared with the proxy storage